<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
   <widget class="FileUploadWidget" name="fileUpload">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Drag CSV</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="dataTable">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileUploadWidget</class>
   <extends>QLineEdit</extends>
   <header location="global">utils</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
